aws/k8s/agent.tf

```diff
@@ -0,0 +1,12 @@
+resource "helm_release" "agent" {
+  depends_on       = [module.my-cluster]
+  name             = "env0-agent"
+  namespace        = "env0-agent"
```

Comment:
You're causing a drift here from our docs, which state `env0-keda`.
However, we're dropping keda soon, so that's probably OK

Reply:
Yes because of that I changed it. I also drop the keda installation